pkg/arvo/sys/hoon.hoon

```diff
@@ -6136,9 +6136,8 @@
         ?.  ?=(%0 -.ref)  ref
         =/  path  $(formula path.formula)
         ?.  ?=(%0 -.path)  path
-        =/  result
-          ?~  scry  ~
-          (scry product.ref product.path)
+        ?~  scry  [%2 trace]
```

Comment:
Let's move this up to the top of `%12` -- *nock* as opposed *mock* should crash as soon as it encounters the `%12`, not after resolving subformulas.